test/Microsoft.Health.Fhir.Ingest.Template.UnitTests/IotJsonPathContentTemplateTests.cs

```diff
@@ -38,17 +39,15 @@ public class IotJsonPathContentTemplateTests
         [FileData(@"TestInput/data_IotHubPayloadExample.json")]
         public void GivenTemplateAndSingleValidToken_WhenGetMeasurements_ThenSingleMeasurementReturned_Test(string eventJson)
         {
-            var evt = EventDataTestHelper.BuildEventFromJson(eventJson);
-            var token = new EventDataWithJsonBodyToJTokenConverter().Convert(evt);
-
+            var token = JsonConvert.DeserializeObject<JToken>(eventJson);

```

Comment:
Why the change from the EventDataTestHelper Event to a JToken?

Reply:
Thanks for reviewing.

Because otherwise, the EventHub dependencies would still be required in this assembly which is not quite necessary for the measurement models. These tests were primarily for the template so this should not reduce the testing coverage by skipping the converters. And in the original package, we still have UTs to cover the conversions.

Reply:
Makes sense. Thanks for clarifying.